README.md

```diff
@@ -1 +1,52 @@
-# math-magician
\ No newline at end of file
+# Leaderboard
+
+This is the first project in the APIs week at Microverse. It is a basic leaderboard with the use of custom Apis from Microverse capstone Project
+
+![screenshot](./images/scn.png)
+
+## Built With
+
+- HTML5
+- CSS
+- Javascript
+- React
+
+## Live Demo
+
+[Live Demo Link](https://DelhinRharl.github.io/math-magician/src/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Install
+
+In your terminal, navigate to your current directory and run this code
+
+`git clone https://github.com/DelhinRharl/math-magician.git`
+
+Locate the directory in your file explorer
+
+`cd Leaderboard`
+
+Install npm or if installed already using this link
+
+`npm install`
+
+Start the web dev server depending on your configuration
+
+`npm start`
+
+The Project should now be live on your browser
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@DelhinRharl](https://github.com/DelhinRharl)
+- Twitter: [@tdelhinrharl](https://twitter.com/delhinrharl)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/AffaxedKiprotich)
+
```

Comment:
- [ ] Good job on your README file information but there are some you have forgotten, please add them :point_down: 

![image](https://user-images.githubusercontent.com/75126481/149460658-c79656c1-29d1-42d9-8137-07264cc7f0b5.png)